src/pyconnect/pyconnectsink.py

```diff
@@ -210,19 +152,16 @@ def all_partitions_at_eof(self):
     def has_partition_assignments(self):
         return len(self._consumer.assignment()) > 0
 
-    @property
-    def last_message(self):
-        # TODO: when bumping to next major release, remove this property
-        warnings.warn(
-            "'last_message' will be permanently renamed to 'current_message' in next major release", DeprecationWarning
-        )
-        return self.current_message
-
     def _run_once(self) -> None:
         try:
             self.current_message = None
             self._status_info = None
-            msg = self._consumer.poll(self.config["poll_timeout"])
+            try:
+                msg = self._consumer.poll(self.config["poll_timeout"])
+            except ConsumeError as ce:
+                logger.debug(f"ConsumeError while polling: {ce.kafka_message}")
+                msg = ce.kafka_message
```

Comment:
Unfortunately they change the logic and raise exception every time kafka produce errors
https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/deserializing_consumer.py#L130

Reply:
Ah I see... can you then raise it again if it is an instance of `SerializationError`?  
Otherwise `msg.error()` will be `None` because it's not a kafka error and the framework won't notice the error which could become an issue because the message hasn't been properly deserialized.